tests/cases/compiler/gettersAndSetters.ts

```diff
@@ -37,3 +37,13 @@ interface I1 {
 }
 
 var i:I1 = function (n) {return n;}
+

```

Comment:
Please add a `// @target: es2020` (or any target newer than es3) at the top of this file. I wish our tests didn't target ES3 by default.

Reply:
yes no problem